gocd/sp4.target.gocd.yaml

```diff
@@ -82,7 +82,8 @@ pipelines:
         tasks:
         - script: |-
             PRODUCTS=$(osc -A https://api.suse.de ls SUSE:SLE-15-SP4:GA | grep "^SLES15-SP4" | grep -v release)
-            for product in kiwi-templates-JeOS SLES-15-SP4-Vagrant sles15-image 000product $PRODUCTS; do
+            set -e
```

Comment:
Move before the `osc` call above?

Reply:
Done 👍

Reply:
You might've forgotten to `git push`

Reply:
Even stupider. I forgot to add before amending ;)